### DISCUSSION TITLE
Multiple listeners on the same method

### DISCUSSION THREAD

**Discussion by skeemer** [livewire/livewire#5912] [1. Help] [answered]
Would it make sense for `Livewire\Attributes\On` to be changed like the following so we can have multiple listeners on the same method? Or did I miss something?
```
#[\Attribute(\Attribute::TARGET_ALL + \Attribute::IS_REPEATABLE)]
class On extends BaseOn
{
    //
}
```

**Comment by joshhanley** [comment, marked as answer]
@skeemer that would make sense to me. There is currently an open PR #5876 to add this to `#[Rule()]` so I've linked this discussion there as it might be able to be added at the same time.

**Reply by skeemer** [reply]
Thank you for following up with this and also to @nuernbergerA and @calebporzio for #5924!
